tests/foreman/ui/test_hostcollection.py

```diff
@@ -690,8 +690,119 @@ def test_positive_install_errata(self):
             self.assertEqual(result, 'success')
             self._validate_package_installed(self.hosts, FAKE_2_CUSTOM_PACKAGE)
 
+    @tier3
+    def test_positive_change_assigned_content(self):
+        """Change Assigned Life cycle environment and content view of host
+        collection
+
+        :id: e426064a-db3d-4a94-822a-fc303defe1f9
+
+        :steps:
+            1. Setup activation key with content view that contain product
+               repositories
+            2. Prepare hosts (minimum 2) and subscribe them to activation key,
+               katello agent must be also installed and running on each host
+            3. Create a host collection and add the hosts to it
+            4. Run "subscription-manager repos" command on each host to notice
+               the repos urls current values
+            5. Create a new life cycle environment
+            6. Create a copy of content view and publish/promote it to the new
+               life cycle environment
+            7. Go to  Hosts => Hosts Collections and select the host collection
+            8. under host collection details tab notice the Actions Area and
+               click on the link
+               "Change assigned Lifecycle Environment or Content View"
+            9. When a dialog box is open, select the new life cycle environment
+               and the new content view
+            10. Click on "Assign" button and click "Yes" button on confirmation
+                dialog when it appears
+            11. After last step the host collection change task page will
+                appear
+            12. Run "subscription-manager refresh" command on each host
+            13. Run "subscription-manager repos" command on each host
+
+        :expectedresults:
+            1. The host collection change task successfully finished
+            2. The "subscription-manager refresh" command successfully executed
+               and "All local data refreshed" message is displayed
+            3. The urls listed by last command "subscription-manager repos" was
+               updated to the new Life cycle environment and content view
+               names
+
+        :BZ: 1315280
+
+        :CaseLevel: System
+        """
+        # create a clients variable here as self.hosts attribute will be
```

Comment:
And how did other tests work in that class? Maybe it will be better to do rename in setup method?

Reply:
other tests user self.hosts before Session

Reply:
But this test user self.hosts that are vms in UI session

Reply:
I open an issue for this https://github.com/SatelliteQE/robottelo/issues/5329